src/app/events/page.tsx

```
@@ -0,0 +1,68 @@
+import React from "react";
+import {
+  handleGetPublicCreatedEvents,
+  handleGetUserCreatedEvents,
+} from "@/actions/event-actions";
+
+import Link from "next/link";
+import CreateEventModal from "@/components/CreateEventModal";
+import CreateEventForm from "@/components/CreateEventForm";
+import { ScrollArea, ScrollBar } from "@/components/ui/scroll-area";
+
+type Props = {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)